src/overlays/actors/ovl_En_Hgo/z_en_hgo.c

```diff
@@ -196,140 +217,161 @@ void EnHgo_DefaultDialogueHandler(EnHgo* this, PlayState* play) {
     if (Message_ShouldAdvance(play)) {
-        switch (this->unk_314) {
+        switch (this->textId) {
             case 0x158F:
                 Message_StartTextbox(play, 0x1590, &this->actor);
-                this->unk_314 = 0x1590;
+                this->textId = 0x1590;
                 break;
+
             case 0x1590:
                 if (gSaveContext.save.weekEventReg[14] & 4) {
                     Message_StartTextbox(play, 0x1591, &this->actor);
-                    this->unk_314 = 0x1591;
+                    this->textId = 0x1591;
                     break;
                 }
+
                 Message_StartTextbox(play, 0x1592, &this->actor);
-                this->unk_314 = 0x1592;
+                this->textId = 0x1592;
                 break;
+
             case 0x1591:
                 Message_StartTextbox(play, 0x1592, &this->actor);
-                this->unk_314 = 0x1592;
+                this->textId = 0x1592;
                 break;
+
             case 0x1593:
                 Message_StartTextbox(play, 0x1594, &this->actor);
-                this->unk_314 = 0x1594;
+                this->textId = 0x1594;
                 break;
+
             case 0x1595:
                 Message_StartTextbox(play, 0x1596, &this->actor);
-                this->unk_314 = 0x1596;
+                this->textId = 0x1596;
                 break;
+
             case 0x1596:
                 Message_StartTextbox(play, 0x1597, &this->actor);
-                this->unk_314 = 0x1597;
+                this->textId = 0x1597;
                 break;
+
             case 0x1598:
                 Message_StartTextbox(play, 0x1599, &this->actor);
-                this->unk_314 = 0x1599;
+                this->textId = 0x1599;
                 break;
+
             case 0x15A5:
                 Message_StartTextbox(play, 0x15A6, &this->actor);
-                this->unk_314 = 0x15A6;
+                this->textId = 0x15A6;
                 break;
+
             case 0x15A6:
                 Message_StartTextbox(play, 0x15A7, &this->actor);
-                this->unk_314 = 0x15A7;
+                this->textId = 0x15A7;
                 break;
+
             case 0x15A7:
                 func_801477B4(play);
-                func_80BD049C(this);
+                EnHgo_SetupTalk(this);
                 break;
         }
     }
 }
 
-s32 func_80BD0898(EnHgo* this, PlayState* play) {
+s32 EnHgo_HandleCsAction(EnHgo* this, PlayState* play) {
     s32 actionIndex;
 
     if (Cutscene_CheckActorAction(play, 486)) {
         actionIndex = Cutscene_GetActorActionIndex(play, 486);
-        if (this->unk_316 != play->csCtx.actorActions[actionIndex]->action) {
-            this->unk_316 = play->csCtx.actorActions[actionIndex]->action;
+        if (this->csAction != play->csCtx.actorActions[actionIndex]->action) {
+            this->csAction = play->csCtx.actorActions[actionIndex]->action;
             switch (play->csCtx.actorActions[actionIndex]->action) {
```

Comment:
I don't suppose using `csAction` itself here matches? Might just look a bit neater.

Reply:
It does not unfortunately